samples/snippets/src/main/java/com/example/monitoring/UptimeSample.java

```diff
@@ -126,7 +126,7 @@ public static void main(String... args) throws IOException {
         listUptimeChecks(projectId);
         break;
       case "listips":
-        listUptimeCheckIPs();
+        listUptimeCheckIps();
```

Comment:
I think it's better to keep same as it is.

Reply:
if this method name remine same then following Checkstyle violation error occur:
```java
 UptimeSample.java:[217,23] (naming) AbbreviationAsWordInName: Abbreviation in name 'listUptimeCheckIPs'
 must contain no more than '1' consecutive capital letters.
 ```